Jasmine/Common/Models/Coordinate.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/// Defines a coordinate system that is zero-indexed, where the top left corner has the coordinate
+/// row = 0, col = 0.
+///
+/// - Author: Wang Xien Dong
```

Comment:
Convention: are we going to write authors on our files? @li-kai @riwu 

I vote no because there's always `git blame`.